apps/discord-bot/src/listeners/events/auto-thread.ts

```diff
@@ -17,11 +17,15 @@ async function autoThread(channelSettings: ChannelWithFlags, message: Message) {
 	if (!isHumanMessage(message)) return;
 	if (message.type !== MessageType.Default) return;
 	if (message.thread) return;
-	// Channel is text based, and message has been sent by a human
+	const authorName = message.member?.nickname ?? message.author.username;
+
+	let textTitle = `${authorName} - ${message.cleanContent}`;
+	if (message.attachments.size > 0 && message.content.length === 0) {
+		textTitle = `${authorName} - ${
```

Comment:
Instead of setting the text title again, set a variable like "thread title" make it equal to clean content and then use the attachment name if it's available

Reply:
Fixed :+1: